pyproject.toml

```diff
@@ -58,6 +58,12 @@ all = [
     'langcheck[ja-optional]',
     'langcheck[zh]'
 ]
+
+# Different LLM clients
+anthropic = [
+    'anthropic'
+]
```

Comment:
I feel like it'll be easier to just include this in the core deps for now, WDYT?

Reply:
I agree it'll be easier, but when I looked into different LLM-based tools with multiple platform types, tools like `llama-index` or `langchain`offer different subpackages for non-OpenAI clients, and `instructor` mark the non-OpenAI dependencies as optional.

It makes sense because most of the users will only use one or two client types, so it is not good to install all the other unused API sdks by default. We need to do this for `gemini` anyways because of the Python version issue.

Therefore, though it is not a strong opinion, I would prefer the current optional dependency style. WDYT?